pkg/cluster/check/discovery.go

```diff
@@ -72,6 +73,10 @@ func memberToNodeInfo(member *clussterres.Member) (cluster.NodeInfo, error) {
 		return cluster.NodeInfo{}, err
 	}
 
+	if len(ips) == 0 {
+		return cluster.NodeInfo{}, fmt.Errorf("no IP address found for member: %s", member.Metadata().String())
```

Comment:
I think `member.Metadata().String()` is too cryptic, we can use `ID()` which should be a hostname/nodename